webapp/channels/src/components/new_channel_modal/new_channel_modal.tsx

```diff
@@ -104,6 +104,8 @@ const NewChannelModal = () => {
     const [canCreateFromPluggable, setCanCreateFromPluggable] = useState(true);
     const [actionFromPluggable, setActionFromPluggable] = useState<((currentTeamId: string, channelId: string) => Promise<Board>) | undefined>(undefined);
 
+    const crypto = require('crypto');
```

Comment:
you can import just like https://github.com/mattermost/mattermost-server/blob/8f9b22b004ca53d0becbce64edf171ae0eda17da/webapp/channels/src/components/admin_console/generated_setting.tsx#L3-L4